### DISCUSSION TITLE
OpenTX to EdgeTX models

### DISCUSSION THREAD

**Discussion by Richjanz** [EdgeTX/edgetx#3149] [Q&A] [answered]
I have recently flashed my Horus x10x express with OpenTx to EdgeTx. I still have a few models from OpenTx that I want to put on  EdgeTx. Will EdgeTx convert them over or will I have to Re-flash the x10s with Edge to get converted?

**Comment by kochpet** [comment, marked as answer]
You only need to open the *.otx file in the EdgeTX Companion, you will be asked if you want to convert the file.

**Reply by chuchotiao** [reply]
Hope you can see this, I just did the migration, I have the .otx file, but I'm unable to open it in EdgeTX Companion (v2.11.0), when trying to open a file I don't have the option to open an OTX file in the Open file Dialog

**Reply by elecpower** [reply]
Companion 2.10 is the last version to support reading otx files. By default multiple Companion versions can be installed side by side. So install v2.10.6 read the otx files and write as etx. Then the etx files can be read by v2.11.
Please carefully check the conversions as not all settings come across as expected due to significant feature differences.

**Reply by chuchotiao** [reply]
Thank you so much! I was thinking something like this but no clue which version, just did what you suggested and I have the models in the radio already
